scripts/restore

```diff
@@ -76,12 +76,19 @@ ynh_system_user_create $app	# Recreate the dedicated user, if not existing
 
 chown -R $app: $final_path
 
+#=================================================
+# RESTORE FAIL2BAN CONFIGURATION
+#=================================================
+ynh_restore_file "/etc/fail2ban/jail.d/$app.conf"
+ynh_restore_file "/etc/fail2ban/filter.d/$app.conf"
+
+systemctl reload fail2ban # Reload instead of restart for better performance
```

Comment:
Use `ynh_systemd_action` instead of systemctl
https://yunohost.org/#/packaging_apps_helpers_en

Reply:
Done :)